client/.env

```diff
@@ -0,0 +1 @@
+PORT=11632
```

Comment:
Should this be 11633?

Reply:
This port is for development only. React-scripts serves to 11632 which then proxies to 11633 for the backend. Deploying production, the backend serves everything on 11633.